runtime/op/meta/pruner.go

```diff
@@ -0,0 +1,27 @@
+package meta
+
+import (
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/order"
+	"github.com/brimdata/zed/runtime/expr"
+)
+
+type Pruner struct {
+	pred expr.Evaluator
+	ectx expr.Context
+}
+
+func NewPruner(e expr.Evaluator, o order.Which) *Pruner {
```

Comment:
Nits: No need to export this. And o is unused.
```suggestion
func newPruner(e expr.Evaluator) *Pruner {
```